src/components/ContinuationIn/ExtraproRegistration.vue

```diff
@@ -653,13 +654,17 @@ export default class ExtraproRegistration extends Mixins(DateMixin) {
     // and we have the home incorporation date (custom component)
     // and we have the affidavit file, if required (custom component)
     // and the other form (Vuetify) components are valid
-    return (
-      this.isBusinessActive &&
+    // show tick mark only when user visits Review Page
+    if (this.getShowErrors) {
+      return (
+        this.isBusinessActive &&
```

Comment:
This is a good solution.

Another way to do this would be to move `getShowErrors` inside the return statement.

Reply:
Thanks! Updated.